conftest.py

```diff
@@ -4,7 +4,17 @@
 
 import pytest
 
-from plugin_runner.plugin_runner import EVENT_HANDLER_MAP, LOADED_PLUGINS, load_plugins
+BASE_DIR = Path("./plugin_runner/tests")
```

Comment:
`.` may mean something different depending on how someone is running tests (terminal, in an IDE run configuration, etc.). Something I sometimes do to provide a little more robustness for test runs is to figure out what the BASE_DIR is relative to the location of a known file. As long as the project structure doesn't change, this should always be the same no matter how someone runs the tests (and if it does change, the tests fail and the fix is straightforward).

```suggestion
BASE_DIR = Path(__file__).parent / "plugin_runner" / "tests"
```